examples/jsm/nodes/shadernode/ShaderNodeBaseElements.js

```diff
@@ -0,0 +1,265 @@
+
+export const normalGeometry = nodeImmutable( NormalNode, NormalNode.GEOMETRY );
+export const normalLocal = nodeImmutable( NormalNode, NormalNode.LOCAL );
+export const normalWorld = nodeImmutable( NormalNode, NormalNode.WORLD );
+export const normalView = nodeImmutable( NormalNode, NormalNode.VIEW );
+export const transformedNormalView = nodeImmutable( VarNode, normalView, 'TransformedNormalView' );
+
+export const viewMatrix = nodeProxy( Object3DNode, Object3DNode.VIEW_MATRIX );
+export const normalMatrix = nodeProxy( Object3DNode, Object3DNode.NORMAL_MATRIX );
+export const worldMatrix = nodeProxy( Object3DNode, Object3DNode.WORLD_MATRIX );
+export const position = nodeProxy( Object3DNode, Object3DNode.POSITION );
+export const viewPosition = nodeProxy( Object3DNode, Object3DNode.VIEW_POSITION );
+
+export const positionGeometry = nodeImmutable( PositionNode, PositionNode.GEOMETRY );
+export const positionLocal = nodeImmutable( PositionNode, PositionNode.LOCAL );
+export const positionWorld = nodeImmutable( PositionNode, PositionNode.WORLD );
+export const positionView = nodeImmutable( PositionNode, PositionNode.VIEW );
+export const positionViewDirection = nodeImmutable( PositionNode, PositionNode.VIEW_DIRECTION );
+
+export const texture = nodeProxy( TextureNode );
+export const sampler = ( texture ) => nodeObject( new ConvertNode( texture.isNode === true ? texture : new TextureNode( texture ), 'sampler' ) );
+export const uv = ( ...params ) => nodeObject( new UVNode( ...params ) );
+export const pointUV = nodeImmutable( PointUVNode );
+
+// gpgpu
+
+export const compute = ( node, count, workgroupSize ) => nodeObject( new ComputeNode( nodeObject( node ), count, workgroupSize ) );
+
+// math
+
+export const EPSILON = float( 1e-6 );
+export const INFINITY = float( 1e6 );
+
+export const cond = nodeProxy( CondNode );
+
+export const add = nodeProxy( OperatorNode, '+' );
+export const sub = nodeProxy( OperatorNode, '-' );
+export const mul = nodeProxy( OperatorNode, '*' );
+export const div = nodeProxy( OperatorNode, '/' );
+export const remainder = nodeProxy( OperatorNode, '%' );
+export const equal = nodeProxy( OperatorNode, '==' );
+export const assign = nodeProxy( OperatorNode, '=' );
+export const lessThan = nodeProxy( OperatorNode, '<' );
+export const greaterThan = nodeProxy( OperatorNode, '>' );
+export const lessThanEqual = nodeProxy( OperatorNode, '<=' );
+export const greaterThanEqual = nodeProxy( OperatorNode, '>=' );
+export const and = nodeProxy( OperatorNode, '&&' );
+export const or = nodeProxy( OperatorNode, '||' );
+export const xor = nodeProxy( OperatorNode, '^^' );
+export const bitAnd = nodeProxy( OperatorNode, '&' );
+export const bitOr = nodeProxy( OperatorNode, '|' );
+export const bitXor = nodeProxy( OperatorNode, '^' );
+export const shiftLeft = nodeProxy( OperatorNode, '<<' );
+export const shiftRight = nodeProxy( OperatorNode, '>>' );
+
+export const rad = nodeProxy( MathNode, MathNode.RAD );
```

Comment:
here too

Reply:
I would note here that `MathNode.RAD` and `MathNode.DEG` were named so even before this PR.

Reply:
At some update this happens and I have not noticed...
@LeviPesin Let use all functions of `GLSL` names for now, and we take any other decision if necessary in the future.

https://github.com/mrdoob/three.js/blob/f3e92d7172534bb23aa461c31c90a2d985c7683f/examples/jsm/nodes/shadernode/ShaderNodeElements.js#L228

Reply:
Done.